Composer/packages/client/src/components/CreationFlow/v2/DefineConversation.tsx

```diff
@@ -126,7 +128,13 @@ const DefineConversationV2: React.FC<DefineConversationProps> = (props) => {
   const normalizeTemplateId = () => {
     if (currentTemplate) {
       // use almost the same patterns as in assetManager.ts
-      return currentTemplate.name.trim().replace(/[-\s]/g, '_').toLocaleLowerCase();
+      const camelCasedName = camelCase(
+        currentTemplate.name
+          .trim()
+          .replace(/bot|maker/gi, '')
```

Comment:
Should this be `replaceAll` instead of `replace`. I think if `currentTemplate.name = 'botMaker'` the result would be `'maker`' instead of an empty string.

Reply:
Spoke about this offline and resolved te issue